core/tests/puppeteer-acceptance-tests/user-utilities/logged-in-users-utils.js

```diff
@@ -0,0 +1,691 @@
+   */
+  async clickContactUsInGetInvolvedMenuOnNavbar() {
+    await this.clickOn(navbarGetInvolvedTab);
+    await Promise.all([
+      this.page.waitForNavigation(),
+      this.clickOn(navbarGetInvolvedTabContactUsButton),
+    ]);
+    if (this.page.url() !== contactUrl) {
+      throw new Error (
+        'The Contact Us button in Get Involved Menu on navbar does not open ' +
+        'the Contact page!');
+    } else {
+      showMessage(
+        'The Contact Us button in Get Involved Menu on navbar opens ' +
+        'the Contact page.');
+    }
+  }
+
+  /**
+   * Function to click the Donate button on navbar
+   * and check if it opens the Donate page.
+   */
+  async clickDonateOnNavbar() {
+    await this.clickOn(navbarDonateButton);
+    if (this.page.url() !== donateUrl) {
+      throw new Error (
+        'The Donate button on navbar does not open the Donate page!');
+    } else {
+      showMessage(
+        'The Donate button on navbar opens the Donate page.');
+    }
+  }
+
+  /**
+   * Function for click the Watch A Video button
+   * in the Thanks for Donating page and check if it opens the right page.
+   */
+  async clickWatchAVideoButtonInThanksForDonatingPage() {
+    await this.page.waitForSelector(watchAVideoButton);
+    const buttonText = await this.page.$eval(
+      watchAVideoButton,
+      element => element.innerText
+    );
+    if (buttonText !== 'Watch a video') {
+      throw new Error (
+        'The Watch A Video button does not exist!');
+    } else {
+      await Promise.all([
+        this.page.waitForNavigation(),
+        this.clickOn(watchAVideoButton),
+      ]);
+      if (this.page.url() !== watchAVideoUrl) {
+        throw new Error (
+          'The Watch A Video button does not open the right page!');
+      } else {
+        showMessage(
+          'The Watch A Video button opens the right page.');
+      }
+    }
+  }
+
+  /**
+   * Function for click the Read Our Blog button
+   * in the Thanks for Donating page and check if it opens the Blog page.
+   */
+  async clickReadOurBlogButtonInThanksForDonatingPage() {
+    await this.page.waitForSelector(readOurBlogButton);
+    const buttonText = await this.page.$eval(
+      readOurBlogButton,
+      element => element.innerText
+    );
+    if (buttonText !== 'Read our blog') {
+      throw new Error (
+        'The Read Our Blog button does not exist!');
+    } else {
+      await Promise.all([
+        this.page.waitForNavigation(),
+        this.clickOn(readOurBlogButton),
+      ]);
+      if (this.page.url() !== blogWebUrl) {
+        throw new Error (
+          'The Read Our Blog button does not open the Blog page!');
+      } else {
+        showMessage(
+          'The Read Our Blog button opens the Blog page.');
+      }
+    }
+  }
+
+  /**
+   * Function to click the dismiss button in the Thanks for Donating page and
+   * check if the Thanks for Donating popup disappear and show the Donate page
```

Comment:
Instead of "show the Donate page", I think "if the Donate page is shown" clarifies that it is a check as opposed to a user interaction.

Also, add a comma after the first "and" *to list the two things the function does*, that being "click" and "check". It also makes it clear that the second "and" is *to list what the function checks for*.